azure-iot-hub-devicesdk/samples/send_telemetry_device.py

```diff
@@ -30,7 +32,13 @@ def connection_state_callback(status):
 # send 5 messages with a 1 second pause between each message
 for i in range(0, 5):
     print("sending message #" + str(i))
-    device_client.send_event("test_payload message " + str(i))
+    msg = Message("test weather payload " + str(i))
+    msg.message_id = uuid.uuid4()
+    msg.user_id = "AZIOT1234"
```

Comment:
>msg.user_id = "AZIOT1234" [](start = 4, length = 25)

nit: I don't think any other SDK uses the user_id property and i'm worried it might be confusing to someone reading the samples (given we also talk of auth providers etc). let's document the property, but not advertise it in samples.